src/Microsoft.VisualStudio.Threading/JoinableTask.cs

```diff
@@ -32,7 +32,7 @@ public partial class JoinableTask : IJoinableTaskDependent
         /// <summary>
         /// Stores the top-most JoinableTask that is completing on the current thread, if any.
         /// </summary>
-        private static readonly ThreadLocal<JoinableTask> CompletingTask = new ThreadLocal<JoinableTask>();
+        private static readonly ThreadLocal<JoinableTask?> CompletingTask = new ThreadLocal<JoinableTask?>();
```

Comment:
💭 I'm a bit surprised this one was needed. Do we *set* this value to null somewhere?

Reply:
We don't need to set it to null. But null is the default value. And I *wanted* everywhere we use `ThreadLocal<T>.Value` to be typed as `JoinableTask?` because it indeed can be null and will always be null unless we set it to something else.

Reply:
> I wanted everywhere we use `ThreadLocal<T>.Value` to be typed as `JoinableTask?`

I'm guessing the netcoreapp3.0 build already did this, but the definition change for the field made it also apply to other cases?

Reply:
Ah, I guess `ThreadLocal<T>.Value` has a `[MaybeNull]` attribute on it. So ya, I guess netcoreapp3.0 didn't require it. But ya, it is more explicit and works for other TFMs this way.